connectors-home/modules/ROOT/pages/introduction/intro-config-oauth2.adoc

```diff
@@ -126,30 +152,30 @@ Here is the XML for the OAuth 2.0 callback configuration for Gmail Connector:
 </gmail:config>
 ----
 +
-. Run the application.
+. Run the app.
 
 === Performing the OAuth Dance
 
-Perform the OAuth dance and authorize your application to access your Gmail account via the connector. The following steps show how to perform the OAuth dance:
+Perform the OAuth dance and authorize your app to access your Gmail account via the connector. The following steps show how to perform the OAuth dance:
 
 . In your browser, navigate to the path specified in the *Authorize Path* field to begin the OAuth dance. In this example, the path is `\http://localhost:8080/authorize`. If everything is configured correctly, you will see a Google Authentication screen in which you select your user account and give permission to the app to access your account. Google shows a list of permissions that require approval, for example:
 +
 image::intro-config-oauth2-gmail-permissions.png[Window that asks the user permission for Gmail to access their account]
 +
-NOTE: These permissions are dependent on the *Scopes* field configured in the Gmail Connector connection configuration. You can accept the defaults, but if you want to limit the access, you can remove the unwanted scopes. For example, remove the `\https://www.googleapis.com/auth/gmail.send` scope to disallow the connector from sending new emails from your account, however, doing so also limits connector functionality, such as the *Send Message* or the *Send Draft* operations.
+NOTE: These permissions are dependent on the *Scopes* field configured in the Gmail Connector connection configuration. You can accept the defaults, but if you want to limit the access, you can remove the unwanted scopes. For example, remove the `\https://www.googleapis.com/auth/gmail.send` scope to disallow the connector from sending new emails from your account, however, doing so limits connector functionality, such as the ability to use the *Send Message* or *Send Draft* operations.
 +
-. Click *Allow*. You are now redirected to `\http://127.0.0.1:8080/callback`, which is the redirect URI that you configured in the OAuth provider and is open in the connector configuration. If the OAuth dance is successful, a `Successfully retrieved access token` message shows up on the browser, which means the connector is now fully authorized and ready to use your Gmail account.
+. Click *Allow*. Mule runtime directs you to `\http://127.0.0.1:8080/callback`, which is the redirect URI that you configured in the OAuth provider and is open in the connector configuration. If the OAuth dance is successful, a `Successfully retrieved access token` message shows up on the browser, which means the connector is now fully authorized and ready to use your Gmail account.
```

Comment:
It's not the mule runtime that redirects the user but the Identity Provider.
Also "redirects" is the correct technical term though this is not a big deal and most people will probably understand either of those.


```suggestion
. Click *Allow*. The Identity provider redirects you to `\http://127.0.0.1:8080/callback`, which is the redirect URI that you configured in the OAuth provider and is open in the connector configuration. If the OAuth dance is successful, a `Successfully retrieved access token` message shows up on the browser, which means the connector is now fully authorized and ready to use your Gmail account.
```